rplugin/python3/deoplete/sources/swift.py

```diff
@@ -62,17 +62,10 @@ def gather_candidates(self, context):
     def identifiers_from_result(self, result):
         out = []
 
-        longest_desc_length = 0
-        for complete in result:
-            desc_len = len(complete['name'])
-
-            if desc_len > longest_desc_length:
-                longest_desc_length = desc_len
-
         for completion in result:
             description = completion['name']
             _type = completion['typeName']
-            abbr = description + ' ' + _type.rjust((len(description) - longest_desc_length) + 1)
+            abbr = '{} {}'.format(description, _type)
```

Comment:
This should be `_type.rjust((longest_sec_length - len(description) + 1)` so that the types are all right-justified at the same point. Unfortunately though the implementation would be broken anyways, as seen in the given screenshot. So removing this for now is definitely better than a broken implementation, and I can go back to it later if need be. Thanks.

<img width="459" alt="screen shot 2016-05-09 at 13 34 48" src="https://cloud.githubusercontent.com/assets/1313522/15127327/06b8c8ee-15eb-11e6-85a4-04612c5faebb.png">